Chrome DevTools UI

tobiasstrebitzer opened the pull request:
I've tweaked the template and stylesheet a little to follow the latest chrome devtools color picker (Spectrum):

- Adjusted various paddings, colors, borders and shadows.
- Adjusted the style of the `picker dot`, so that it renders opaque and is visible beyond the color reel bounds.

tobiasstrebitzer commented:
Really great job on this colorpicker @narsenico, i've been looking for something that comes closed to the devtools color picker for a long time now :-)

narsenico commented:
Hi @tobiasstrebitzer,
you did a really good job, new ui is fantastic.

I've prepared a pre-release `npm i a-color-picker@chrome`
this is the demo
[https://codepen.io/narsenico/full/VVEEaZ](https://codepen.io/narsenico/full/VVEEaZ)

Next week I'll merge all into master branch.

tobiasstrebitzer commented:
I've implemented a couple more changes / fixes:

- **Update palette style so that 10 colors fit into one row:** This is due to color palettes often serving 10 colors.
- **Fix input changes not being applied:** I noticed the input change handler was still trying to get the property from the `name` field, changed that to `nameref`.
- **destroy:** Implemented a `destroy()` method in the public interface, useful for on-deman / popup color pickers.
- **Color reel position calculation:** Updated the `findColor(r, g, b)` method to use HSV values, which make it easier to calculate target x/y coordinates.

![colorpicker-changes](https://user-images.githubusercontent.com/222509/49202002-1414d900-f3de-11e8-8657-ca6fbf16cac7.png)

narsenico commented:
all changes merged

Thanks again 👍
